### PR TITLE
Add add, remove and update functions

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, I made functions for add item, remove item, and update item

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hello @vladanvi99**, 

Good job so far!

Nice job @vladanvi99 on adding the drag and drop, but there is just a small thing to check comments below 👇

## Required Changes ♻️
- The remove completed does remove all the (completed & non-completed items) so let's dive to correct it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/addRemoveUpdateClear.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  arr.forEach((item) => {
+    item.addEventListener('mouseover', (event) => onMouseOver(event, item));
+    item.addEventListener('mouseout', (event) => onMouseOut(event, item));
+    item.addEventListener('dragstart', (event) => onDragStart(event, item));
+    item.addEventListener('drop', (event) => onDragEnd(event, item, arr, listWrap));
+  });
+  const removeBtns = document.querySelectorAll('.remove-item');
+  removeBtns.forEach((item) => item.addEventListener('click', () => removeItem(item, arr, container)));
+  const toDoChecksInputs = [...document.querySelectorAll('.list-item .check-item')];
+  toDoChecksInputs.forEach((item) => item.addEventListener('change', () => checkItem(item)));
```
Comment (dasileker) [inline src/addRemoveUpdateClear.js:43]: - [ ] You can add a loop to check if the item is checked or completed if it's true remove.

File: src/index.js [hunk 2]
```diff
@@ -62,6 +53,17 @@ toDoItems.forEach((item) => {
...
-  item.addEventListener('drop', (event) => onDragEnd(event, item, toDoArr));
+  item.addEventListener('drop', (event) => onDragEnd(event, item, toDoItems, listWrap));
 });
 listWrap.addEventListener('dragover', (event) => onDragOver(event, listWrap));
+// ADD ITEM
+const addNewButton = document.querySelector('.add-new-btn');
+addNewButton.addEventListener('click', (event) => addNewItem(event, toDoArr, toDoItems, listWrap));
+// CLEAR ALL
+const clearAllBtn = document.querySelector('.clear-all');
+clearAllBtn.addEventListener('click', () => clearAll(listWrap));
```
Comment (dasileker) [inline src/index.js:64]: - [ ]  Or you target the check input of the item to set it clear, we do want just the one checked or completed to be removed.

Comment (vladanvi99) [reply src/index.js:64]: Thank you very much, I will fix it 😃

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**